src/e3/testsuite/__init__.py

```diff
@@ -1247,11 +1245,7 @@ def write_comment_file(self, comment_file: IO[str]) -> None:
         # Sensible default: just write the command line used to run the
         # testsuite. Testsuites can override or reuse this.
         quoted_cmdline = " ".join(quote_arg(arg) for arg in sys.argv)
-        comment_file.write(
-            f"Testsuite options:"
-            f"\n  {quoted_cmdline}"
-            f"\n"
-        )
+        comment_file.write(f"Testsuite options:" f"\n  {quoted_cmdline}" f"\n")
```

Comment:
There’s no need to have separate string literals now, please merge them

Reply:
Fixed. Thanks for the catch.

Reply:
Thanks!